docs/rst/developer_manual/installation/sources/linux.rst

```diff
@@ -217,6 +217,10 @@ For example, on Ubuntu use the command:
 
    sudo apt install liblz4-dev libzstd-dev
 
+.. note::
+
+    To work with MCAP files via command line interface, you can use `MCAP CLI <https://mcap.dev/guides/cli>` _ to manage the data in MCAP files.
```

Comment:
i thought we do not want to `MCAP CLI` usage and details in eProsima doc here for the maintenance. instead, we can introduce that there is `MCAP CLI` that we can use to manage the saved MCAP files via command line interface here.